### ISSUE TITLE
Quoted Identifiers in Column Names

### ISSUE THREAD

**Issue by lapter57** [xnuinside/simple-ddl-parser#55]
Hi! If  I use of quoted identifiers in the names of table columns,  then return `[]`.
```sql
CREATE TABLE IF NOT EXISTS shema.table
(
    field_1        BIGINT,
    `partition`   STRING,
);
```

**Comment by xnuinside** [comment]
@lapter57  hi, yes, right now supported form only with "" quotes & without, I will add support for `

**Comment by xnuinside** [comment]
@lapter57 https://github.com/xnuinside/simple-ddl-parser/pull/56/files released, test added, version 0.19.2 with fix https://pypi.org/project/simple-ddl-parser/
